tests/test-cases/Group24-Host-Affinity/24-01-Basic.md

```diff
@@ -0,0 +1,40 @@
+Suite 24-01 - Basic
+===================
+
+# Purpose:
+To verify basic VM-Host Affinity functionality
+
+# References:
+1. [The design document](../../../doc/design/host-affinity.md)
+
+# Environment:
+This test requires a vCenter Server environment where VCHs can be deployed and container VMs created.
+
+
+Positive Testing
+----------------
+
+### 1. Creating a VCH creates a VM group and container VMs get added to it
+
+#### Test Steps:
+1. Verify that no DRS VM Group exists by the expected name.
+2. Create a VCH.
+3. Verify that a DRS VM Group was created and that the endpoint VM was added to it.
+4. Create a variety of containers.
+5. Verify that the container VMs were added to the DRS VM Group.
```

Comment:
we could use a naming convention here as well to ensure all VMs in the group are the VMs we expect.
Only really relevent if we're able to test two VCHs updating different VM groups at the same time.

Reply:
I think it's worth having a test along those lines, but perhaps in a separate test (or suite?). Added a comment to the relevant issue to ensure this idea isn't lost: https://github.com/vmware/vic/issues/7558#issuecomment-376295957